package.json

```diff
@@ -21,7 +21,7 @@
     "check-types": "lerna run check-types",
     "check-spelling": "pyspelling -c .pyspelling.yml -v",
     "docs:install": "pip install --user -r docs/requirements.txt",
-    "docs:build": "lerna run check-readme && lerna run docs:build && ./scripts/prepare-docs.sh",
+    "docs:build": "lerna run check-readme && lerna run docs:build --scope @chainsafe/lodestar && ./scripts/prepare-docs.sh",
```

Comment:
doesn't lerna skip all packages anyways that do not have the command `docs:build` defined?

Reply:
I ran `time` command on the two commands and I don't notice any difference in terms of performance and output:
```
time lerna run docs:build                            
info cli using local version of lerna
lerna notice cli v7.3.0

> @chainsafe/lodestar:"docs:build"

@chainsafe/lodestar: yarn run v1.22.19
$ node --loader ts-node/esm ./docsgen/index.ts
@chainsafe/lodestar: (node:48908) ExperimentalWarning: `--experimental-loader` may be removed in the future; instead use `register()`:
@chainsafe/lodestar: --import 'data:text/javascript,import { register } from "node:module"; import { pathToFileURL } from "node:url"; register("ts-node/esm", pathToFileURL("./"));'
@chainsafe/lodestar: (Use `node --trace-warnings ...` to show where the warning was created)
@chainsafe/lodestar: Done in 2.16s.

 —————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————

 >  Lerna (powered by Nx)   Successfully ran target docs:build for project @chainsafe/lodestar


lerna run docs:build  3.44s user 0.67s system 145% cpu 2.817 total
```
```
time lerna run docs:build --scope @chainsafe/lodestar
info cli using local version of lerna
lerna notice cli v7.3.0
lerna notice filter including "@chainsafe/lodestar"
lerna info filter [ '@chainsafe/lodestar' ]

> @chainsafe/lodestar:"docs:build"

@chainsafe/lodestar: yarn run v1.22.19
$ node --loader ts-node/esm ./docsgen/index.ts
@chainsafe/lodestar: (node:48897) ExperimentalWarning: `--experimental-loader` may be removed in the future; instead use `register()`:
@chainsafe/lodestar: --import 'data:text/javascript,import { register } from "node:module"; import { pathToFileURL } from "node:url"; register("ts-node/esm", pathToFileURL("./"));'
@chainsafe/lodestar: (Use `node --trace-warnings ...` to show where the warning was created)
@chainsafe/lodestar: Done in 2.20s.

 —————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————

 >  Lerna (powered by Nx)   Successfully ran target docs:build for project @chainsafe/lodestar


lerna run docs:build --scope @chainsafe/lodestar  3.45s user 0.76s system 125% cpu 3.355 total
```

> doesn't lerna skip all packages anyways that do not have the command docs:build defined?
That's my understanding as well.

Reply:
Hmm I might have been confused by the already long output :)

Still looks weird to use lerna here, as it's a script that only makes sense for `cli`, as opposed to something like `test`.